gremlin-python/src/main/python/tests/process/test_traversal.py

```diff
@@ -31,7 +31,7 @@
 from gremlin_python.process.traversal import Binding, Bindings
 from gremlin_python.process.graph_traversal import __
 
-gremlin_server_url = 'ws://localhost:{}/gremlin'
+gremlin_server_url = (os.environ['GREMLIN_SERVER_URL']) if 'GREMLIN_SERVER_URL' in os.environ else 'ws://localhost:{}/gremlin'
```

Comment:
can we avoid copying this line of code several times?

Reply:
I suppose i can import the variable from `conftest.py`, is that conventional for python?